03/app.js

```diff
@@ -0,0 +1,12 @@
+const btns = document.querySelectorAll('button');
+
+const callback = function(e) {
+    if(this.textContent !== "clicked") {
+        console.log('clicked');
+        this.textContent = "clicked";
+    }
```

Comment:
To rozwiązanie tylko połowicznie rozwiązuje problem.
"callback" jest uruchamianay tylko nie widać efektu. 
Lepszym rozwiązaniem będzie usunięcie nasłuchiwania.